pages/about.html

```diff
@@ -44,26 +44,31 @@
         <section class="home-section about-section">
             <div class="home-section-desktop-container">
                 <div class="home-section-overlay">
-                    <div class="search-task-card about-us">
-                        <h1 class="app-moto underline-center">About Us</h1>
+                    <h1 class="about-us-title underline-center">About Us</h1>
+                    <div class="about-us-content">
```

Comment:
- [ ] [OPTIONAL] Kindly make sure to fix the About page design in the mobile view, as for now the header section text is not displaying properly, if your using a fixed height make sure to keep in mind the mobile version

![Screenshot_20230601_100119](https://github.com/habitoz/serategna/assets/25227022/2c8625f3-cf76-4fbc-b406-e781d5bd479d)